ios/Exponent/Kernel/Views/EXAppViewController.m

```diff
@@ -77,6 +85,12 @@ - (void)dealloc
 - (void)viewDidLoad
 {
   [super viewDidLoad];
+  
+  if (![EXEnvironment sharedEnvironment].isDetached
+      && _appRecord != [EXKernel sharedInstance].appRegistry.homeAppRecord) {
```

Comment:
would be great to actually show this when loading home in development mode -- i think we do this currently(?). do we need the second condition here if production mode apps never send the progress event?

Reply:
Will double check 👍
I'm relieved that you seem to be very familiar with how it should work (in terms what is presented and when) 😌